docs/source/setup-guide.rst

```diff
@@ -69,26 +65,27 @@ Step 5: Setup python virtual environment + setup
 
 Python virtualenv ensures that you have the correct python and modules needed by this code.
 
-If you don't have ``virtualenv``, install it; for example, on a Debian-based Linux such as Ubuntu, use:
+If you don't have the python ``venv`` module, install it; for example, on a Debian-based Linux such as Ubuntu, use:
 
 .. code-block:: bash
 
-   $ apt install python3-virtualenv
```

Comment:
I'm pretty sure that the virtual env is not needed at all any more, so long as all python is run using scripts/pyrun. I'll try this out.

Reply:
I haven't been using either virtualenv or venv, and it's been working for me.   I think the actual reason why it was added in the docs was to get python3.7 instead of 3.9, in cases where 3.9 is the system default.

In any case, venv is preferable to virtualenv since `-m venv` is in the standard Python library, to my understanding.

Reply:
> so long as all python is run using scripts/pyrun

It seems that at least the proj `Makefile` is launching its python-based rules with this:
https://github.com/google/CFU-Playground/blob/main/proj/proj.mk#L101

Are we sure that this cascade down to all python invocations of the lower layer of the build process?